internal/locker_test.go

```diff
@@ -90,7 +90,7 @@ func TestAccLock(t *testing.T) {
 				indexOfAnInactiveLocker = i
 			}
 			assert.ErrorContains(t, lockerErrs[i], "lock acquired by")
-			assert.ErrorContains(t, lockerErrs[i], "Use --force to override")
```

Comment:
why was it --force and not --force-deploy before?

Reply:
We named it force before recent changes that introduced git validation logic and renamed the flag to `--force-deploy` for deploy and `--force-lock` for destroy